class RecordNotFoundException(Exception):
    pass


class RecordAlreadyExistsException(Exception):
    pass
